lusidtools/lpt/lpt.py

```diff
@@ -50,9 +50,9 @@ def access(obj, col):
 
     # Try standard representations
     try:
-        iterator = iter(data)
-    except:
         iterator = iter(data.values)
+    except:
```

Comment:
to check

Reply:
In some cases data is an iterable with a values field - we want to use this field rather than data